react/ui-components/src/atoms/TextInput.js

```diff
@@ -358,7 +377,11 @@ const TextInput = (props) => {
                 }
               }}
               ref={props.inputRef}
-              value={props?.value}
+              value={
+                props.type === "date"
+                  ? formatDateReadable(props?.value)
```

Comment:
can you add extra flag for date, let default be current way and if they send that flag format to this new format @NabeelAyubee